components/landing/Contact.vue

```diff
@@ -17,24 +17,7 @@
 import { Component, Vue } from 'nuxt-property-decorator'
 
 
-@Component<Contact>({
-  metaInfo() {
-    return {
-      title: 'KodaDOT cares about environmental impact',
-      titleTemplate: '%s | Low Carbon NFTs',
-      meta: [
-        { name: 'description', content: 'Creating Carbonless NFTs on Kusama' },
-        { property: 'og:title', content: 'KodaDOT cares about environmental impact'},
-        { property: 'og:image', content: 'https://nft.kodadot.xyz/kodadot_carbonless.jpg'},
-        { property: 'og:description', content: 'Creating Carbonless NFTs on Kusama'},
-        { property: 'twitter:title', content: 'KodaDOT cares about environmental impact'},
-        { property: 'twitter:description', content: 'Creating Carbonless NFTs on Kusama'},
-        { property: 'twitter:image', content: 'https://nft.kodadot.xyz/kodadot_carbonless.jpg'},
-      ]
-    }
-  },
-  components: {}
-})
+@Component<Contact>({})
```

Comment:
```suggestion
@Component<Contact>
```

Reply:
this doesn't work. `@Component` need at least 1 argument for its constructor, even if its empty